perfume-api/src/main/java/io/perfume/api/sample/domain/Sample.java

```diff
@@ -0,0 +1,71 @@
+package io.perfume.api.sample.domain;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDateTime;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.time.LocalDateTime' import. Should be before 'org.jetbrains.annotations.NotNull'.